<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CityTypeInfoDialog</class>
 <widget class="QWidget" name="CityTypeInfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>190</width>
    <height>184</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CityTypeInfoDialog</string>
  </property>
  <widget class="QLabel" name="labelId">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>ID:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditId">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditName">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelType">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Type:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditType">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelType_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>MapId:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditMapId">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
